friends/src/App.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import './App.css';
+import FriendList from './Component/FriendList';
+import axios from 'axios';
+
+class App extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      friends: [],
+      id: '',
```

Comment:
`id` is handled by the back-end for this project, so you don't have to worry about it 🎉